src/shell/shell_cmds.cpp

```diff
@@ -240,16 +238,36 @@ void DOS_Shell::CMD_HELP(char * args){
 	HELP("HELP");
 	bool optall=ScanCMDBool(args,"ALL");
```

Comment:
Can this become a `const`? (I don't think it's updated below?)

Reply:
Sure, it can be a const and I updated it accordingly. It is also a bool rather than const bool in other shell commands but I guess they can be modified in separate PRs too.